### PR TITLE
update JuMP names

### PR DESCRIPTION
**Pull request by chriscoey**
i can't run the examples because .jl files are in the gitignore file, @joehuchette

### CODE REVIEW

File: examples/branch_jump.jl [hunk 1]
```diff
@@ -16,17 +16,17 @@ u = MathProgBase.getconstrUB(m_internal)
...
 # populate JuMP model with data from internal model
-@defVar(mod, x[1:n])
+@variable(mod, x[1:n])
 for i in 1:n
-    setLower(x[i], xlb[i])
-    setUpper(x[i], xub[i])
-    (vtypes[i] == :Bin || vtypes[i] == :Int) ? mod.colCat[x[i].col] = :Int : nothing # change vartype to integer when appropriate
+    setlower(x[i], xlb[i])
+    setupper(x[i], xub[i])
+    (vtypes[i] == :Bin || vtypes[i] == :Int) ? mod.colcat[x[i].col] = :Int : nothing # change vartype to integer when appropriate
```
Comment (joehuchette) [inline examples/branch_jump.jl]: I don't think this actually changed?

File: src/JuMPfunctions.jl [hunk 2]
```diff
@@ -21,7 +21,7 @@ function initcb(m::JuMP.Model)
 end
 
 function solvehook(m::JuMP.Model; kwargs...)
-    JuMP.buildInternalModel(m)
+    JuMP.buildinternalmodel(m)
```
Comment (joehuchette) [inline src/JuMPfunctions.jl]: ``build`` I believe

File: src/JuMPfunctions.jl [hunk 3]
```diff
@@ -32,7 +32,7 @@ function solvehook(m::JuMP.Model; kwargs...)
             end
             m.ext[:cb].branchcallback(d)
         end
-        setbranchcallback!(m.internalModel, branchcallback)
+        setbranchcallback!(m.internalmodel, branchcallback)
```
Comment (joehuchette) [inline src/JuMPfunctions.jl]: not changed

File: src/JuMPfunctions.jl [hunk 4]
```diff
@@ -41,24 +41,24 @@ function solvehook(m::JuMP.Model; kwargs...)
             m.colVal = copy(d.sol)
             m.ext[:cb].incumbentcallback(d)
         end
-        setincumbentcallback!(m.internalModel, incumbentcallback)
+        setincumbentcallback!(m.internalmodel, incumbentcallback)
```
Comment (joehuchette) [inline src/JuMPfunctions.jl]: ditto

File: test/solvetime_01.jl [hunk 5]
```diff
@@ -14,11 +14,11 @@ using CPLEX
...
-@addConstraint(m, x + 2*y + z <= 15)
+@variable(m, 5 >= x >= 0)
+@variable(m, 10 >= y >= 0, Int)
+@variable(m, z, Bin)
+@objective(m, Max, x + 2*y + 5*z)
+@constraint(m, x + y + z <= 10)
+@constraint(m, x + 2*y + z <= 15)
 status = solve(m)
-println("walltime = $(MathProgBase.getsolvetime(getInternalModel(m)))")
+println("walltime = $(MathProgBase.getsolvetime(getinternalmodel(m)))")
```
Comment (joehuchette) [inline test/solvetime_01.jl]: ``internalmodel``

**Comment by joehuchette** [comment]
Huh? Which .gitignore

**Comment by joehuchette** [comment]
Hmm I can't track it down. I don't think there's anything particular to that file (or any behavior to expect, besides the callback running), so any MIP in an .lp file should do.

**Comment by chriscoey** [comment]
ok well maybe you should comment it to reflect what your intentions for the file were, or delete it

**Pull request merged**